src/policy.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2022 IBM
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use crate::db;
+use anyhow::*;
+use serde::Deserialize;
+use std::fs;
+
+const DEFAULT_POLICY_PATH: &str = "default_policy.json";
```

Comment:
consider adding `--policy=<filename>` command-line argument with a default value of `default_policy.json`.  This will allow users to override.

Reply:
let's do this once we merge given that it is technically a new feature

Reply:
I agree.